drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -2690,6 +2690,11 @@ void RasterizerSceneGLES3::_render_list_template(RenderListParameters *p_params,
 								glBlendFuncSeparate(GL_DST_COLOR, GL_ZERO, GL_ZERO, GL_ONE);
 							}
 
+						} break;
+						case GLES3::SceneShaderData::BLEND_MODE_PREMUL_ALPHA: {
```

Comment:
```suggestion
						case GLES3::SceneShaderData::BLEND_MODE_PREMULT_ALPHA: {
```
I prefer PREMUL (without the "t"), but CanvasItems use PREMULT, so we should use that everywhere :/

Reply:
Yeah :/ i updated it but i missed some.